README.md

```diff
@@ -601,13 +589,13 @@ once they have been initially created on disk.
 * **shift**: As mentioned in the "Normal Mode and Add-Node Mode" section above, quickly pressing and releasing the
 shift key will toggle between "normal mode" and "add-node mode".  See the section below
 entitled "Advanced Topic: Control Area: Settings" for information on how to change this
-behavioor.
+behavior.
 
 * **delete** and **backspace**: If you place the cursor over a node, so that the node turns cyan,
-you can delete the node by pressing either of these two keys.  As mentioned above, khartes does
-not have an "undo" function, so think carefully before doing this.
+you can delete the node by pressing either of these two keys.  As mentioned above, deleting a node
+will delete it permanently, so think carefully before doing this.
```

Comment:
```diff
- you can delete the node by pressing either of these two keys.  As mentioned above, khartes does
- not have an "undo" function, so think carefully before doing this.
+ you can delete the node by pressing either of these two keys.  As mentioned above, deleted nodes
+ can not be recovered, so think carefully before doing this.
```

See https://github.com/KhartesViewer/khartes/pull/17#discussion_r1381921243 .

Reply:
Same comment as before; I want to emphasize the general lack of an undo function.

Reply:
I would suggest to avoid duplicate content. This refers back to the first mention, in which it can already be handled.
See https://github.com/KhartesViewer/khartes/pull/17#discussion_r1382369773

Reply:
That seems fine.